src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/ParenPadCheck.java

```diff
@@ -306,6 +299,15 @@ private static int[] makeAcceptableTokens() {
             TokenTypes.SUPER_CTOR_CALL,
             TokenTypes.LAMBDA,
             TokenTypes.RECORD_DEF,
+            TokenTypes.TYPECAST,
+            TokenTypes.STAR,
+            TokenTypes.PLUS,
+            TokenTypes.MINUS,
+            TokenTypes.DIV,
+            TokenTypes.MOD,
+            TokenTypes.LAND,
+            TokenTypes.LOR,
+            TokenTypes.LNOT,
```

Comment:
add some tokens that got missed due to having no deep scan (we may need to add some more tokens but those were the obvious one from the test and input files)